typescript/packages/dash-components/package.json

```diff
@@ -1,26 +1,32 @@
 {
-  "name": "@webviz/dash-components",
-  "version": "0.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "Equinor <opensource@equinor.com>",
-  "license": "MPL-2.0",
-  "dependencies": {
-    "@equinor/eds-core-react": "^0.12.1",
-    "@equinor/eds-icons": "^0.19.1",
-    "@mui/material": "^5.12.2",
-    "@webviz/core-components": "^0.6.2",
-    "convert-units": "^2.3.4",
-    "d3": "^7.8.2",
-    "lodash": "^4.17.21",
-    "mathjs": "^9.4.2",
-    "uuid": "^8.3.2"
-  },
-  "devDependencies": {
-    "deep-equal": "^2.0.5",
-    "jsverify": "^0.8.4"
-  }
+    "name": "@webviz/dash-components",
+    "version": "0.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "Equinor <opensource@equinor.com>",
+    "license": "MPL-2.0",
+    "dependencies": {
+        "@equinor/eds-core-react": "^0.12.1",
+        "@equinor/eds-icons": "^0.19.1",
+        "@mui/material": "^5.12.2",
```

Comment:
Suggest moving to `peerDependencies`. Consumers may want to use different potentially conflicting versions.